src/content/desci/index.md

```diff
@@ -102,11 +102,13 @@ Explore projects and join the DeSci community.
 - [Fleming Protocol: open-source data economy that fuels collaborative biomedical discovery](https://medium.com/@FlemingProtocol/a-data-economy-for-patient-driven-biomedical-innovation-9d56bf63d3dd)
 - [Active Inference Lab](https://www.activeinference.org/)
 - [Cherubs DAO: NFT funded and DAO governed impact fund for mental health & brain wellness](https://cherubsdao.com/)
+- [SCINET, INC: decentralized life sciences ("DeSci") research and investment platform building dapps to speed up scientific research from lab to life](https://www.scinet.one/)
```

Comment:
It also doesn't really seem related to Ethereum, the only blockchain mentioned is ICP?

Reply:
This project seems to be closed-source and the website is very light on detail. I personally wouldn't endorse it on the DeSci page because it is impossible to judge how decentralized it is or how it enables open science.

Reply:
OK. I see what you're saying. I have a few questions though in response:
1. Is the page exclusively for Ethereum projects? If it is, then I understand @minimalsm's concern about ICP and I'll drop it right there. 

2. But in case it's not limited to Ethereum projects...regarding @minimalsm's and @jmcook1186's other points, for one, all projects on ICP are completely decentralized, much more so than Ethereum which relies on storage on AWS etc. All front- and back-end for ICP projects are decentralized. More specific to whether it's DeSci, there is similar ambiguity in many of the other "Get Involved" pages. It is not clear how people can get involved with Atoms. Molecule does the same thing as we do regarding funding Desci research, except they are limited to drug discovery and we do it for the entire life sciences ecosystem. Opsci has less content on their page than we do, or the same. Fleming Protocol just cites a Medium article. I'm not ragging on any of these projects at all, as we admire, talk to, and work with many of them. However, it seems like someone of an inconsistent standard at this point. I hope you can see my side of things on this.

Reply:
In addition, we're referenced by some of the articles listed on the page in Further reading as a DeSci project.

Specifically in Jocelynn's DeSci wiki, Sarah Hamburg's Guide to Desci. Also, interviewed and referenced in Cointelegraph's  "DeSci: Can crypto improve scientific research?" (https://cointelegraph.com/magazine/2022/04/15/desci-tokens-help-improve-scientific-research). Incidentally that would also be a good Further Reading.

Reply:
Decentralization is about more than L1 architecture, which is why transparency about the org and ts processes is important. But you raise a fair point about our criteria for linking out to desci projects - we need to define them more clearly.

Pinging @minimalsm @samajammin @wackerow for their thoughts.

Reply:
agree